apps/music/js/ui/views/player_view.js

```diff
@@ -228,19 +230,19 @@ var PlayerView = {
         this.album.classList.remove('hidden-cover-share');
       }
     } else {
-      var titleBar = document.getElementById('title-text');
+      var titleBar = TitleBar.titleText;
```

Comment:
@hfiguiere Although it's not related to the RTL issue, I think we should use `TitleBar.changeTitleText()` here to change the title, but additionally we need to modify it to be able to set l10nId as well. This is probably something we didn't find before while the music refactoring part 1, I am fine with doing it in this commit or file a followup so feel free to modify it or not.